### ISSUE TITLE
battle count is sometimes wrong

### ISSUE THREAD

**Issue by frozenpandaman** [frozenpandaman/splatnet2statink#37]
```
Checking to see if there are unuploaded battles before exiting...
No remaining battles found.
4 wins and 1 loss this session.
4 wins and 1 loss against the other Splatfest team.
2 mirror matches against your Splatfest team.
Bye!
```

```
Checking to see if there are unuploaded battles before exiting...
No remaining battles found.
4 wins and 0 losses this session.
4 wins and 0 losses against the other Splatfest team.
1 mirror match against your Splatfest team.
Bye!
```

**Comment by frozenpandaman** [comment]
Made a (hopeful) fix. Will find a way to test it tomorrow…

**Comment by frozenpandaman** [comment]
issue was also with regular battle count…
```
Checking to see if there are unuploaded battles before exiting...
New battle result detected at 8:49:14 PM! (Academy, Won)
Battle uploaded to https://stat.ink/@frozenpandaman/spl2/439453
Successfully uploaded remaining battles.
0 wins and 1 loss this session.
Bye!
```

cause was capitalization issue:
https://github.com/frozenpandaman/splatnet2statink/blob/941ee52542db630ea9e3dcf6785e8c7dcc25d7e2/splatnet2statink.py#L377
(we do it this way instead of a bool because we print out the variable later)

fixed in f7c58a9:
https://github.com/frozenpandaman/splatnet2statink/blob/f7c58a957f882bf75a8b9047ff67de01caf9acd8/splatnet2statink.py#L381
